xbmc/GUIInfoManager.cpp

```diff
@@ -188,6 +188,18 @@ typedef struct
   int  val;
 } infomap;
 
+const infomap string_bools[] =   {{ "isempty",          STRING_IS_EMPTY },
+                                  { "equals",           STRING_EQUALS },
+                                  { "startswith",       STRING_STARTS_WITH },
+                                  { "endswith",         STRING_ENDS_WITH },
+                                  { "contains",         STRING_CONTAINS }};
+
+const infomap integer_bools[] =  {{ "equals",           INTEGER_EQUALS },
+                                  { "greater",          INTEGER_GREATER_THAN },
+                                  { "greaterorequals",  INTEGER_GREATER_OR_EQUALS },
+                                  { "less",             INTEGER_LESS_THAN },
+                                  { "lessorequals",     INTEGER_LESS_OR_EQUALS }};
```

Comment:
> you are able to negate the expression by adding ! in front

Well that you can say about all of them. You don't *need* 5 operators, you need less, but It's about having shorthand is it not? IMO we should follow convention: https://en.wikipedia.org/wiki/Relational_operator#Standard_relational_operators